railties/lib/rails/tasks/framework.rake

```diff
@@ -64,5 +64,18 @@ namespace :rails do
     task :bin do
       RailsUpdate.invoke_from_app_generator :create_bin_files
     end
+
+    task :schema_migrations => :environment do
+      if ActiveRecord::SchemaMigration.table_exists?
```

Comment:
Active Record may not be present here and it will fail. We need to check its presence first.